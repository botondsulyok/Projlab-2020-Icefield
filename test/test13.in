field stable 0
field hole 1 true 1
neighbours stable hole
neighbours hole stable
entity mover Eskimo stable
stats stable
stats hole
stats mover
move mover hole
stats stable
stats hole
stats mover