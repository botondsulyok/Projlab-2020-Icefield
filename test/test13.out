id: stable
type: iceunit
building: plains0
snow: 0
pickable: null
standers: mover
pullers: mover
masters: mover
neighbours: hole

id: hole
type: unstable
building: plains1
snow: 1
pickable: null
standers: null
pullers: null
masters: null
neighbours: stable
capacity: 1
broken: true

id: mover
type: eskimo
iceUnit: stable
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

Game Over
id: stable
type: iceunit
building: plains0
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: hole

id: hole
type: unstable
building: plains1
snow: 0
pickable: null
standers: mover
pullers: mover
masters: mover
neighbours: stable
capacity: 1
broken: true

id: mover
type: eskimo
iceUnit: hole
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

